### PR TITLE
Feature/add model for comments

### PR DESCRIPTION
**Pull request by slevvvvin**

### CODE REVIEW

File: blog/templates/post_details.html [hunk 1]
```diff
@@ -12,7 +11,26 @@ <h1>{{ post.title }}</h1>
             <a href="{% url 'delete_post' post.pk %}">(Delete)</a>
         {% endif %}
     </small>
-    <hr>
-    {{ post.body|safe }}
+    <div class="post-detail-body">
+        {{ post.body|safe }}
+    </div>
+    <h2>Comments</h2>
+    {% if not post.comments.all %}
```
Comment (TinaArts) [inline blog/templates/post_details.html]: Use 'for … empty' construction for show "No comments yet..." message

File: blog/urls.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 from django.urls import path
 from .views import HomeView, PostDetailView, AddPostView, UpdatePostView,\
-    DeletePostView, AddCategoryView, CategoryView, UserRegisterView
+    DeletePostView, AddCategoryView, CategoryView, UserRegisterView, \
```
Comment (TinaArts) [inline blog/urls.py]: For more readable code when you have many classed for import use tuple

File: blog/views.py [hunk 3]
```diff
@@ -57,3 +57,15 @@ class DeletePostView(DeleteView):
     model = Post
     template_name = 'delete_post.html'
     success_url = reverse_lazy('home')
+
+
+class AddCommentView(CreateView):
+    model = Comment
+    form_class = AddCommentForm
+    template_name = 'add_comment.html'
+    success_url = reverse_lazy('home')
```
Comment (TinaArts) [inline blog/views.py]: Why user will go to home page after added comment to post?

File: blog/helper/middleware.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+
+def get_current_user():
+    return _user.value if hasattr(_user, 'value') else None
+
+
+def get_current_user_id():
+    current_user = get_current_user()
+    return current_user.id if current_user and \
+                              current_user.is_authenticated() else 0
```
Comment (TinaArts) [inline blog/helper/middleware.py]: Please remove unused file

**Review by TinaArts — CHANGES_REQUESTED** [review]

File: blog/helper/middleware.py (hunk 4 above)
Comment (slevvvvin) [reply blog/helper/middleware.py]: done

File: blog/views.py (hunk 3 above)
Comment (slevvvvin) [reply blog/views.py]: fixing this with redirection to current post page

File: blog/urls.py (hunk 2 above)
Comment (slevvvvin) [reply blog/urls.py]: done

File: blog/templates/post_details.html (hunk 1 above)
Comment (slevvvvin) [reply blog/templates/post_details.html]: done

**Review by TinaArts — APPROVED** [review]

**Pull request merged**